.prettierignore

```diff
@@ -1,8 +1,19 @@
 # Add files here to ignore them from prettier formatting
+.git-blame-ignore-revs
+.editorconfig
+.npmrc
+LICENSE
 
 /dist
 /coverage
+/env
+/.husky
```

Comment:
This counters the setting you have in `prettier.config.mjs` where we indicate that files under `.husky/*` pattern should be formatted with shellscript parser. Instead, add this:
```suggestion
/.husky/_/
```